doc/source/visualizing/plots.rst

```diff
@@ -922,54 +925,137 @@ Use any of the colormaps listed in the :ref:`colormaps` section.
    slc.set_cmap(("gas", "density"), "RdBu_r")
    slc.save()
 
-The :meth:`~yt.visualization.plot_window.AxisAlignedSlicePlot.set_log` function
-accepts a field name and a boolean.  If the boolean is ``True``, the colormap
-for the field will be log scaled.  If it is ``False`` the colormap will be
-linear.
+Colorbar norms
```

Comment:
I think I was mostly concerned with keeping the term from matplotlib's API, but I think you are right that this doesn't resonate with most users. I'm fine with renaming this section "Colorbar Normalization / Scaling", and maybe I'll add a link to matplotlib's docs to establish the correspondence.

Reply:
About replacing all instances of "norm" with another word, I'm not so sure about it. The problem is that I'm introducing the `PlotContainer.set_norm` method, and I really want to keep that name because it aligns with matplotlib's naming convention for scaling classes (e.g., `LogNorm`), which users need to know about if they're going to use this method.
For now I'm going to make it explicit that "norm" and "scaling" are synonymous in this text.

Reply:
(although I could also rename the method to `set_normalization` if you think that's preferable)

Reply:
Looks good.  Thanks!